stytch/config/version.go

```diff
@@ -1,3 +1,3 @@
 package config
 
-const APIVersion = "11.5.2"
+const APIVersion = "11.6.0"
```

Comment:
Definitely a MAJOR update, I'm going to cut a v12 branch to prepare a few things

Reply:
Okay, let's have this merge into `v12-prerelease` branch